### DISCUSSION TITLE
Possible to redact download client user:password in logs?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#256] [Q&A]
Since logs are stored in plain text, is it possible to redact sensitive info like download client login info and any API keys?

**Comment by mmgoodnow** [comment]
This _should_ already be happening: https://github.com/mmgoodnow/cross-seed/blob/f7502f27982fdcf1242959bd0c35d85420722b8c/src/logger.ts#L26-L56

Are you having trouble? Which option is your sensitive info leaking from?

**Reply by SawkeeReemo** [reply]
Not having trouble, I was just poking around in the logs and saw on the error logs when I tested to see if I could inject qBT via the reverse proxy I have it behind (I could not). When it errored out, it put the full string in plain text with my qBT password in there. I just found it by chance.

**Reply by mmgoodnow** [reply]
Does your qBT password contain a question mark?

**Reply by SawkeeReemo** [reply]
It does not. But it is a randomized jumbled mess generated by a password vault’s password generator. There are a few non-letter/number characters in there.
